Description: Use smaller example for integralClosure documentation
 Otherwise, we get memory errors on i386 and mipsel.
Bug: https://github.com/Macaulay2/M2/issues/1423
Author: Doug Torrance <dtorrance@piedmont.edu>
Applied-Upstream: 1.17, https://github.com/Macaulay2/M2/commit/b3e971b
Last-Update: 2020-10-26

--- a/M2/Macaulay2/packages/IntegralClosure.m2
+++ b/M2/Macaulay2/packages/IntegralClosure.m2
@@ -1343,40 +1343,40 @@
      to instead compute Hom(J^-1, J^-1).  This is usually a more time consuming
      computation, but it does potentially get to the answer in a smaller number of steps.
    Example
-     S = QQ[x,y]
-     f = ideal (y^4-2*x^3*y^2-4*x^5*y+x^6-x^7)
+     S = QQ[x,y,z]
+     f = ideal (x^8-z^6-y^2*z^4-z^3)
      R = S/f
      time R' = integralClosure R
      netList (ideal R')_*
      icFractions R
    Example
-     S = QQ[x,y]
-     f = ideal (y^4-2*x^3*y^2-4*x^5*y+x^6-x^7)
+     S = QQ[x,y,z]
+     f = ideal (x^8-z^6-y^2*z^4-z^3)
      R = S/f
      time R' = integralClosure(R, Strategy => Radical)
      netList (ideal R')_*
      icFractions R
    Example
-     S = QQ[x,y]
-     f = ideal (y^4-2*x^3*y^2-4*x^5*y+x^6-x^7)
+     S = QQ[x,y,z]
+     f = ideal (x^8-z^6-y^2*z^4-z^3)
      R = S/f
      time R' = integralClosure(R, Strategy => AllCodimensions)
      netList (ideal R')_*
    Example
-     S = QQ[x,y]
-     f = ideal (y^4-2*x^3*y^2-4*x^5*y+x^6-x^7)
+     S = QQ[x,y,z]
+     f = ideal (x^8-z^6-y^2*z^4-z^3)
      R = S/f
      time R' = integralClosure(R, Strategy => SimplifyFractions)
      netList (ideal R')_*     
    Example
-     S = QQ[x,y]
-     f = ideal (y^4-2*x^3*y^2-4*x^5*y+x^6-x^7)
+     S = QQ[x,y,z]
+     f = ideal (x^8-z^6-y^2*z^4-z^3)
      R = S/f
      time R' = integralClosure (R, Strategy => RadicalCodim1)
      netList (ideal R')_*
    Example
-     S = QQ[x,y]
-     f = ideal (y^4-2*x^3*y^2-4*x^5*y+x^6-x^7)
+     S = QQ[x,y,z]
+     f = ideal (x^8-z^6-y^2*z^4-z^3)
      R = S/f
      time R' = integralClosure (R, Strategy => Vasconcelos)
      netList (ideal R')_*
